config/config.go

```diff
@@ -0,0 +1,137 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
```

Comment:
I don't think that more config file formats are a good thing. I believe it's the opposite, because it just brings in more code and confusion, when everybody uses different files, which are harder to compare. I don't see the need for that. All the formats are, for our purposes here, simple enough so that everybody can learn to use them and personal tastes shouldn't matter. I didn't even know, for example, something like TOML exists and I have no idea why somebody thought the world needs another one of those

Reply:
I will roll this back and remove the viper dep